routes/web.php

```diff
@@ -217,7 +217,7 @@ function () : void {
                     "hero" => [
                         "hero_logo" => "/images/logo_tc_colour.png",
                         "hero_logo_alt" => "The GC Talent Cloud Graphic Identifier.",
```

Comment:
This should probably be localized as well.

Reply:
hero alt text localization a1d30b5